bip-0044.mediawiki

```diff
@@ -275,6 +275,7 @@ is required and a pull request to the above file should be created.
 * [[https://www.ledgerwallet.com/|Ledger Wallet]] ([[https://github.com/LedgerHQ|source]])
 * [[https://21.co/learn/21-lib-wallet/|21 Machine Wallet]] ([[https://github.com/21dotco|source]])
 * [[https://trustwalletapp.com/|Trust Wallet]] ([[http://github.com/trustWallet/|source]])
+* [[https://horizontalsystems.io/dapps/bank-wallet/ | BANK Wallet]] ([[https://github.com/horizontalsystems/bank-wallet-ios/|iOS Sources ]]) / ([[https://github.com/horizontalsystems/bank-wallet-android/|Android Sources ]])
```

Comment:
Please remove extra spaces: before closing bracket `]`, before and after the pipe `|` -- they result in a broken link.

Reply:
The other wallets in this list use "source" (singular, lowercase).
For consistency, please also use "source" rather than "_Sources_".